### DISCUSSION TITLE
Possibility of using pixel TileMatrixSet

### DISCUSSION THREAD

**Discussion by nadavFux** [developmentseed/titiler#666] [Q&A] [answered]
Hi, 
I am considering using titiler to replace an existing tiler service I have in use, **but I am not sure if titiler fits my needs**-

have a dataset of images which are **not georeferenced**(at least not accurately),
in my application when a call is made to tile the map, a calculation runs that returns the accurate crop location of the selected image and a request is sent to a similar tiler **(WMS/WMTS) but with a TileMatrixSet which is a plain pixel set** (the bbox is the exact pixels window to tile)

seeing how the TileMatrixSet of titler is base on rio-tiler which is based on morecantile Im left unsure **whether titiler could fit my needs**,
I don't have extensive knowledge of titiler so I hope to be wrong in this and that there is a way to "skip" the georeferenced being done in the TileMatrixSet handling of the tiling process.

I would much appreciate any answer that will help me better understand what my possibilities are in this situation 
and any possible solutions that will fit the needs I describe as I really do have to find an alternative to my existing tiler...

thanks in advance :)

**Comment by vincentsarago** [comment, marked as answer]
👋 @nadavFux 

We've been working on a titiler plugin specifically for non-geo image https://github.com/developmentseed/titiler-image

It's still a WIP because I was waiting on some update in the IIIF specification and also needed to fix the DeepZoom endpoints

**Comment by nadavFux** [comment]
Thank you for the reply! 
the titiler-image repo has been very helpful, especially now that it's out of WIP

closing.
